pkgs/development/compilers/llvm/10/clang/default.nix

```diff
@@ -8,7 +8,7 @@ let
     pname = "clang";
     inherit version;
 
-    src = fetch "clang" "1w7ixr16a9f0g5kv4irvhwq973wn0d418kb0p9rabyfscm05wfmq";
+    src = fetch "clang" "19y2nii0rqq8hf657d331s8rmpddlm5wvcznhdwxkymd0fd5l7vm";
```

Comment:
in line 39: `# 10.0.0rc3-only` --> `# 10.0.0rc4-only`

Reply:
I changed the comment to `10.0.0 only, this should be present in 10.0.1`, since this is not rc-specific.